content/Store Support/Shipping/About-Ascendia.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "About Ascendia"
+---
+
+Asendia is an available shipping method for the Arduino US store. You can choose this option for most orders inside and outside the United States.
+
+Beware that even though this is the cheapest shipping option, it has the longest delivery time (similar to local post delivery times). For shipments outside the U.S., the delivery can take up to one month. Also, there can be issues tracking and locating your parcel.
+
+> Note that some countries create a local tracking number for these shipments. The local tracking number can be found on the right side of the Ascendia order receipt. This number is valid as soon as the package arrives in the destination country. ![Ascendia order receipt with local tracking number highlighted](img/AscendiaLocalTrackingNumber.png)
```

Comment:
```suggestion
> Note that some countries create a local tracking number for these shipments. The local tracking number can be found on the right side of the Asendia order receipt. This number is valid as soon as the package arrives in the destination country. ![Asendia order receipt with local tracking number highlighted](img/AscendiaLocalTrackingNumber.png)
```

Reply:
Maybe you also could rename the image file.

Reply:
Will do. Thanks for the review

Reply:
[9b00f49](https://github.com/arduino/help-center-content/pull/30/commits/9b00f49ceb42c838b2941aa4282795b6adabdcee)